### ISSUE TITLE
Filtering out an SC level leaves behind the SC numbers and short handles for the filtered out content

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-wcag-quickref#200]
A blind user contacted me to tell me the checkboxes on the left side were not working because when he filtered out AAA he still heard them with his Screen Reader... here's our discussion via email and his recommendation..

**DAROLD** ( a user)

Can you tell me where to report an issue with the Filters on the WCAG 2.1 How to meet quick reference page. Using the latest Google Chrome 78 here, clearing the level AAA checkbox in the Levels filter does not remove the level AAA results from the page for me.
Thanks,

**David**
I just checked it. It seems to remove the AAA success criteria full text on keyboard and mouse click of the checkbox. However, as a reference they leave behind the AAA number and the short name of the Success criteria...  Also visually those numbers and short names are made small and lighter text so they are not as visually obvious... but there is no way to do that for a screen reader. Would you like to file a bug that this is confusing and would rather they leave out the numbers and short names completely? Or put some little screen reader note around them to say they are only there for reference?

**Darold**
This clears it up for me, and yes it is confusing. What they are doing makes no sense to me. In any results filtering that I know of, all references to the filtered out material are removed from the updated results page. I also believe this was how it worked on the WCAG 2.0 quick reference page.
I am recommending that all numbers and related text be removed from the page for the filtered out results. So, when users select not to display the Level AAA  success criteria, all AAA number and short title text will not be displayed on the updated page.

**Recommendation**
Remove filtered out content completely OR put a simpler placeholder of something like 
>>"AAA Success Criteria have been filtered out." 

rather than listing those that were filtered out.

**Comment by yatil** [comment]
Hi @DavidMacDonald,

thanks for the feedback. The behavior was explicitly requested by EOWG or AGWG (I cannot remember which) as it would be confusing if the SCs just disappeared. (That was the reasoning back then, I don’t necessarily agree)

We do have a notice/live region that says: “Selected Filters: WCAG 2.1: success criteria for levels A and AA and all techniques.” which is what you are suggesting. (Changing the wording would mean having it read “Selected Filters: WCAG 2.1: success criteria for levels AA and AAA filtered out and displaying all techniques.” which would make it significantly harder to parse in most circumstances.)

I could add an `aria-hidden="true"` to the SCs but that would interfere with the left hand navigation that always displays all SCs (again, by WG request). Another idea would be to add invisible text “Filtered Out:” to the SC heading when it is filtered out.

**Comment by iadawn** [comment]
No response from original poster
